atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -320,9 +327,21 @@ def set_group_manifest_info(self, extra, worker_metadatas):
 
                                 repositories.append(pullspec)
 
+                            annotations = get_worker_build_info(self.workflow, platform).\
+                                build.get_annotations()
+
                             instance['extra']['docker']['repositories'] = repositories
                             self.log.debug("reset tags to so that docker is %s",
                                            instance['extra']['docker'])
+                            instance['extra']['docker']['digests'] = {}
```

Comment:
Could have a local variable for this dict until it's finished, then assign it to instance[...][...][...] to save repeated look-ups.

Reply:
Fixed